tests/dummy/app/pods/docs/advanced/server-configuration/template.md

```diff
@@ -1,22 +1,13 @@
 # Server configuration
 
-The MirageJS server is configured for you by ember-cli-mirage. However, if you
```

Comment:
@cah-briangantzler minor, thinking if would be more helpful to say `The MirageJS server used to be configured for you by ember-cli-mirage.`